config.json

```diff
@@ -456,6 +456,17 @@
         ],
         "difficulty": 2
       },
+      {
+        "slug": "resistor-color-trio",
+        "name": "Resistor Color Trio",
+        "uuid": "e8e6c84e-4982-4bb0-af49-fd67e6f32920",
+        "practices": [],
+        "prerequisites": [
+          "arrays",
+          "enums"
```

Comment:
This exercise doesn't seem to use enums, so I'm not sure it should be a prerequisite.

Actually, now that I double-check it, the same is true for the `resistor-color-duo` exercise. The example implementation happens to use an enum, but that is by no means the only way to solve it.

Could you remove the `enums` concept from the prerequisites for both exercises?

Reply:
Sure!